tests/test_integration.py

```diff
@@ -361,10 +363,11 @@ class MockServerRequestHandler(MyTestHandler):
 
         stripe.api_base = "http://localhost:%s" % self.mock_server_port
 
-        resp = await stripe.raw_request_async(
+        client = StripeClient("sk_test")
```

Comment:
I think the general pattern in tests is to create the client once and re-use it across the different test cases. Can you confirm that by looking at any other test file and make the changes appropriately?

Reply:
It seems to be a mixed bag: `test_generated_examples.py` and `test_integration.py` create a new client per test, `test_oauth.py` uses an in-class fixture, and a few other files (`test_webhook.py`, `test_file_upload.py`) use a fixture defined in `conftest.py`.  Do we have a preferred pattern?

Reply:
@xavdid-stripe Any preferences?

Reply:
am ok with doing that at a later time

Reply:
Since I was adding StripeClient lines as part of this PR, if that's the ultimate direction we want to go I'm inclined to use the fixture here.

Reply:
quick update: I made the change to use the fixtures in test_raw_request, but for test_integration many of the tests override the default options in some way, so I kept the per-test client pattern and made sure the constructor args were correctly set up.